stubs/redis/redis/cluster.pyi

```diff
@@ -215,23 +217,27 @@ class ClusterPipeline(RedisCluster[_StrType], Generic[_StrType]):
     def script_load_for_pipeline(self, *args, **kwargs) -> None: ...
     def delete(self, *names): ...
 
-def block_pipeline_command(name: str) -> Callable[..., Any]: ...
+def block_pipeline_command(name: str) -> Callable[..., NoReturn]: ...
 
 class PipelineCommand:
-    args: Any
-    options: Any
-    position: Any
-    result: Any
-    node: Any
+    args: Sequence[EncodableT]
+    options: _ParseResponseOptions
+    position: int | None
+    result: Any | None
```

Comment:
Looks like this is often some kind of exception:
- https://github.com/redis/redis-py/blob/b5ebada842c715e9ee74ea638a7e6d11afcddae1/redis/cluster.py#L2242
- https://github.com/redis/redis-py/blob/b5ebada842c715e9ee74ea638a7e6d11afcddae1/redis/cluster.py#L2272
- https://github.com/redis/redis-py/blob/b5ebada842c715e9ee74ea638a7e6d11afcddae1/redis/cluster.py#L2275

```suggestion
    result: Any | Exception | None
```